tools/depends/xbmc-addons.include

```diff
@@ -1,6 +1,7 @@
 ADDON_DEPS_DIR := $(BUILDDIR)/$(PLATFORM)/build/depends
 TOOLCHAIN_FILE = $(ADDON_DEPS_DIR)/share/Toolchain_binaddons.cmake
 ADDON_PROJECT_DIR = $(BUILDDIR)/../../../../cmake/addons
+export PKG_CONFIG_LIBDIR = $(BUILDDIR)/$(PLATFORM)/build/depends/lib/pkgconfig
```

Comment:
please use $(ADDON_DEPS_DIR)/lib/pkgconfig, it's more readable
Then it's good to go

Reply:
Great, should be ready now.